fdbclient/Knobs.cpp

```diff
@@ -172,6 +172,7 @@ void ClientKnobs::initialize(bool randomize) {
 	init( BACKUP_STATUS_DELAY,                    40.0 );
 	init( BACKUP_STATUS_JITTER,                   0.05 );
 	init( MIN_CLEANUP_SECONDS,                  3600.0 );
+	init( RESTORE_IGNORE_LOG_FILES,              false ); // When set to true, the log files in restore will be ignored.
```

Comment:
Can you move the comment to the header file? The comment can be something like: `Default is false. When set to true, the log files will be ignored during the restore, which can produce inconsistent restored data.`

Reply:
Done, thanks.